app/Models/Labels/Sheet.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+namespace App\Models\Labels;
+
+abstract class Sheet extends Label
```

Comment:
To me it doesn't seem especially intuitive that a sheet is an extension of a single label. To me it feels like composition may be a cleaner solution here by making `Sheet` is its own standalone class that represents a composition of many  `Label` objects

Reply:
I actually wrote it that way in my initial testing, then found I was repeating the same code again.
I've made it this way so theoretically any label can be included as a template inside any other label.

Reply:
what code was being repeated?

Maybe I'm too new to this codebase or thinking too high level to see why this didn't work, but couldn't you have the Sheet define the margins and spacing for the sheet (like the word templates do with the tables in the Avery template word docs) and then call a function in each label within the sheet to have it render itself and plonk that into its table square in the sheet to build the whole sheet?